### PR TITLE
Fixes, updates and additions for general file formats table

### PR DESCRIPTION
**Pull request by Argent77**

### CODE REVIEW

File: file_formats/general.htm [hunk 1]
```diff
@@ -127,7 +127,14 @@
               <td><code>.acm</code></td>
               <td>???</td>
               <td>&nbsp;</td>
-              <td>Music.</td>
+              <td>Music. ACM compressed sound files.</td>
+            </tr>
+            <tr>
+              <td>N/A</td>
+              <td><code>.acm</code></td>
```
Comment (lynxlynxlynx) [inline file_formats/general.htm:134]: Really, not ogg?

File: file_formats/general.htm [hunk 2]
```diff
@@ -148,16 +155,23 @@
               <td><code>.mve</code></td>
               <td>???</td>
               <td>&nbsp;</td>
-              <td>Movies. Proprietry Interplay format.</td>
+              <td>Movie. Proprietry Interplay format.</td>
```
Comment (lynxlynxlynx) [inline file_formats/general.htm:158]:
```suggestion
              <td>Movie. Proprietary Interplay format.</td>
```

File: file_formats/general.htm (hunk 1 above)
Comment (Argent77) [reply file_formats/general.htm:134]: No, `.acm` file extension is shared by both ACM and Ogg encoded sound. Ogg encoding is used primarily by PST:EE music.

**Comment by lynxlynxlynx** [comment]
Thanks!

**Pull request merged**